invokeai/backend/stable_diffusion/diffusion/cross_attention_control.py

```diff
@@ -265,7 +265,7 @@ def einsum_op_mps_v1(self, q, k, v):
         if q.shape[1] <= 4096:  # (512x512) max q.shape[1]: 4096
             return self.einsum_lowest_level(q, k, v, None, None, None)
         else:
-            slice_size = math.floor(2 ** 30 / (q.shape[0] * q.shape[1]))
+            slice_size = math.floor(2**30 / (q.shape[0] * q.shape[1]))
```

Comment:
Is there a reason to not use a hex literal? E.g. 0x40000000

Reply:
I'm not the author but that is like an order of magnitude less clear than `2**30` for a smooth brain like me :D